Editor/MonoDevelop.Xml.Editor.csproj

```diff
@@ -19,14 +19,23 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Core\MonoDevelop.Xml.Core.csproj" />
-    <PackageReference Include="Microsoft.VisualStudio.ImageCatalog" Version="16.9.31023.347" ExcludeAssets="runtime" />
-    <PackageReference Include="Microsoft.VisualStudio.Language.Intellisense" Version="16.9.227" ExcludeAssets="runtime" />
-    <PackageReference Include="Microsoft.VisualStudio.Language.StandardClassification" Version="16.9.227" ExcludeAssets="runtime" />
-    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="16.10.56" ExcludeAssets="runtime" />
+    <PackageReference Include="Microsoft.VisualStudio.ImageCatalog" Version="17.1.32107.28" ExcludeAssets="runtime" />
+    <PackageReference Include="Microsoft.VisualStudio.Language" Version="17.1.387" ExcludeAssets="runtime" />
+    <PackageReference Include="Microsoft.VisualStudio.Language.Intellisense" Version="17.1.387" ExcludeAssets="runtime" />
+    <PackageReference Include="Microsoft.VisualStudio.Language.StandardClassification" Version="17.1.387" ExcludeAssets="runtime" />
+    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="17.1.46" ExcludeAssets="runtime" />
   </ItemGroup>
 
   <ItemGroup>
     <Reference Include="System.ComponentModel.Composition" />
   </ItemGroup>
 
+  <!-- Turn off windows dependency check. Some packages still have windows-specific dependencies, even though in the cases that matter they are not used at run-time -->
+  <Target Name="IgnoreWindowsReferenceError" BeforeTargets="_CheckForTransitiveWindowsDesktopDependencies">
```

Comment:
This target should only be necessary when a `net6.0*` project references `net472` nupkgs. In this project, every package _should_ resolve as `netstandard2.0` for the `net6.0` target framework, but for some reason NuGet is resolving them as `net472`, triggering this error and also https://github.com/NuGet/Home/issues/5957.

Reply:
I should note that this happens even if I remove the `net472` from `TargetFrameworks` completely.